lib/Doctrine/ORM/Repository/InvalidFindByInverseAssociation.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Doctrine\ORM\Repository;
+
+use Doctrine\ORM\RepositoryException;
+
+final class InvalidFindByInverseAssociation extends \BadMethodCallException implements RepositoryException
```

Comment:
I'm not a huge fan of this class name. I know I created it myself, but it's derived from the former method name. What I don't like is that it kind of implies there could be valid find by inverse association method calls. Should I drop the `Invalid` part?

Reply:
I guess the name can be discussed in combination with the exception methods. I suggest `InvalidAssociation` or `InvalidAssociationSide`. I've read the exception message in the tests and used the mentioned names for it.

Reply:
`InvalidAssociationSide` sounds great

Reply:
But the other one leaves more room for other named constructors.